utils/tangleid/CoreAPI.js

```diff
@@ -9,22 +15,31 @@ class CoreAPI {
    * @param {string} settings.provider - An address that provide the TanlgeID API.
    */
   constructor(settings) {
-    this.provider = settings.provider;
+    this.provider_local = settings.provider_local;
+    this.provider_swarm = settings.provider_swarm;
   }
 
   fetchUserList() {
-    return fetch(`${this.provider}/fetchUserList`)
-      .then(response => response.json());
+    return fetch(`${this.provider_local}/fetchUserList`)
+      .then(handleErrors)
+      .then(response => response.json())
+      .catch((error) => {
+        console.error(error);
+      });
   }
 
   fetchUserInfo(uuid) {
-    return fetch(`${this.provider}/user/${uuid}`)
-      .then(response => response.json());
+    return fetch(`${this.provider_local}/user/${uuid}`)
+      .then(handleErrors)
+      .then(response => response.json())
+      .catch((error) => {
+        console.error(error);
+      });
   }
 
   fetchClaims(uuid) {
     // forward request to Backend API
-    return fetch(`${this.provider}/proxy/`, {
```

Comment:
You need to mention the removal of `/proxy/` within Git commit messages since it is not trivial.

Reply:
Added.